docs/manual/forms/json_schema.rst

```diff
@@ -0,0 +1,99 @@
+.. _manual_json_schema:
+
+JSON-schema genereren
+=====================
+Het is mogelijk om een `JSON-schema`_ van een formulier te genereren. Momenteel kan dit voor de Objecten API- of de
+Generieke JSON-registratie. Het schema beschrijft de inzendgegevens van alle component- en door de gebruiker
+gedefinieerde variabelen, alsof deze via een van deze plugins zijn verzonden. De eigenschapsnamen van de de componenten
+worden als sleutel gebruikt in de ``properties`` van het schema. Voor componenten worden de volgende
+configuratieopties (indien gespecificeerd) opgenomen in het schema:
+
+ - Label
+ - Beschrijving
+ - Validatieregels, bijvoorbeeld maximale lengte of een regex-patroon
+
+Hoe werkt het
+-------------
+Om een JSON-schema van een formulier te genereren, ga naar het tabblad **Registratie** en configureer een
+:doc:`Objecten API- </configuration/registration/objects>` of
```

Comment:
please use `:ref:` instead where you point to the label: https://github.com/open-formulieren/open-forms/blob/34c8023497c0bc49dab35a5e6729d4a6aff60e11/docs/configuration/registration/objects.rst?plain=1#L1

```suggestion
:ref:`Objecten API- <configuration_registration_objects>` of
```